### DISCUSSION TITLE
Possible to run Paperless for multiple users?

### DISCUSSION THREAD

**Discussion by marclar** [jonaswinkler/paperless-ng#765] [Support] [answered]
I'm building a NAS for my family and I'd like to have Paperless consume documents for each user _without_ sharing those documents between family members.

Each user will have their own account on the NAS, so I could create a "consume" directory in each user's root directory, but how can Paperless distinguish and assign incoming documents to the correct user?

I realize I could have each family member manually drop files onto the web UI, but I'd rather have them mount a drive, configure their scanner to dump files into their mount, and have Paperless handle it from there.

(This is within a `docker-compose.yml`, fwiw).

Thanks!

**Comment by shamoon** [comment, marked as answer]
Multi-user setup is not supported yet but is in the roadmap (at some point), see https://github.com/jonaswinkler/paperless-ng/discussions/52

Edit: up-vote that one if you like!
